makefile

```diff
@@ -1,9 +1,17 @@
-init_composer:
+composer.phar:
 	curl -s https://getcomposer.org/installer | php
 	php composer.phar install --prefer-dist -o --dev
 
+.PHONY: test
 test:
 	vendor/bin/phpunit -c tests/ tests/
 
-build: init_composer test
+.PHONY: analyse
+analyse: phpstan.phar
+	php -r 'exit(PHP_VERSION_ID > 70000 ? 0 : 1);' && ./phpstan.phar analyse -c phpstan.neon -a vendor/autoload.php -l 7 src || echo "skipping analysis"
```

Comment:
now the build will pass if phpstan fails, right? The `|| echo "skipping analysis"` will take precedence?